src/webportal/src/app/job-submission/utils/utils.js

```diff
@@ -218,3 +218,11 @@ export function getStoragePlugin(extras) {
   }
   return plugins.find(plugin => plugin.plugin === 'teamwise_storage');
 }
+
+export function getDeshuttleStorageDetails(dshuttleMountPoint) {
```

Comment:
The method only valid when ufs type is Azure. Can we change the return to {ufsType, accountName, containerName} when ufs type is Azure. And return {ufsType, ufsUri} when UFS type is others. And show difference in web UI

Reply:
Done.